src/index.html

```diff
@@ -2,13 +2,63 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img src="images/logo.png"
+             alt="MOYO logo"
+             class="logo"
+        >
```

Comment:
```suggestion
        <img 
          src="images/logo.png"
          alt="MOYO logo"
          class="logo"
        >
```
[Checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) 13
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/37bca041-6a2a-4bc1-8e55-eab5b9e45ba3)

Reply:
Thanks, have fixed